packages/api/src/external/commonwell/shared.ts

```diff
@@ -32,3 +33,22 @@ export function buildCwOrgName({
   }
   return `${vendorName} - ${orgName}`;
 }
+
+export function getCwPatientContactType(
+  telecom: Contact[] | null | undefined,
+  system: "phone" | "email"
+): string[] {
+  if (telecom && telecom.length > 0) {
+    const contacts: string[] = [];
```

Comment:
nit: if we move this before the if, we can have a single `return contacts;` at the end after the if block